### DISCUSSION TITLE
Tap <Space+s+t > show live grep,  how to retain the search text when tap once again

### DISCUSSION THREAD

**Discussion by kaneruan** [LunarVim/LunarVim#3274] [General]
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/13065289/196347048-695fba68-b9af-415f-ae2f-72f9ec8c6094.png">

**Comment by Leolainen** [comment]
Do you want something like `:Telescope resume`? 
I added this to my config to quickly re-open my last search results
`lvim.builtin.which_key.mappings["s"]["l"] = { "<cmd>Telescope resume<CR>", "Last search" }`

**Comment by bncpr** [comment]
You can press `<C-p>` for previous searches. This might be `<C-k>` in the default configuration, I think.
